uaclient/files/state_files.py

```diff
@@ -295,3 +295,12 @@ def __init__(self, attached_at: datetime.datetime):
     defaults.DEFAULT_PRIVATE_DATA_DIR,
     private=True,
 )
+
+
+def delete_state_files():
+    machine_id_file.delete()
+    status_cache_file.delete()
+    attachment_data_file.delete()
+    anbox_cloud_credentials_file.delete()
+    reboot_cmd_marker_file.delete()
+    services_once_enabled_file.delete()
```

Comment:
I don't think we want to delete services_once_enabled

Reply:
Good point, let me update that